quickbooks_desktop_endpoint.rb

```diff
@@ -172,7 +172,8 @@ def add_flow_return_payload
 
     {
       id: @payload[type][:id],
-      external_guid: @payload[type][:external_guid]
+      external_guid: @payload[type][:external_guid],
+      key: ['external_guid']
```

Comment:
well, it creates when adding it the first time in qbe.... :thinking: it wont exist in flowlink yet, this is for updates flows ?

Reply:
it will still use id the first time

Reply:
Yeah, but we know that we haven't gotten the item from QBE yet, so the id will be the same. The id field is updated when we get from QBE.